### PR TITLE
VirusTotal packs: Update support to VT team

### PR DESCRIPTION
**Pull request by danipv**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/VirusTotal-Private_API/README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-Note: Support for this pack will be moving to the partner around September 28, 2021.
```
Comment (mchasepan) [inline Packs/VirusTotal-Private_API/README.md:1]: Please don't delete this line; instead:

Note: Support for this pack moved to the partner on Oct 1, 2021. Please contact the partner directly via the support link on the right.

File: Packs/VirusTotal/README.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
-Note: Support for this pack will be moving to the partner around September 28, 2021.
```
Comment (mchasepan) [inline Packs/VirusTotal/README.md:1]: Please don't delete this line; instead:

Note: Support for this pack moved to the partner on Oct 1, 2021. Please contact the partner directly via the support link on the right.

File: Packs/VirusTotal/pack_metadata.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
     "name": "VirusTotal",
     "description": "Analyze suspicious hashes, URLs, domains and IP addresses",
-    "support": "xsoar",
-    "currentVersion": "2.1.8",
+    "support": "partner",
+    "currentVersion": "2.1.9",
     "author": "Cortex XSOAR",
```
Comment (mchasepan) [inline Packs/VirusTotal/pack_metadata.json:6]:
```suggestion
    "author": "VirusTotal",
```

File: Packs/VirusTotal/pack_metadata.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
     "url": "https://www.paloaltonetworks.com/cortex",
```
Comment (mchasepan) [inline Packs/VirusTotal/pack_metadata.json:7]:
```suggestion
    "url": "https://www.virustotal.com",
```

**Review by abaumgarten — APPROVED** [review]

**Pull request merged**